opennft/opennft.py

```diff
@@ -1776,6 +1776,7 @@ def selectRoi(self):
 
         if self.P['Type'] == 'PSC':
             if not os.path.isdir(self.P['RoiFilesFolder']):
+                logger.error("Couldn't find: " + self.P['RoiFilesFolder'])
```

Comment:
Please, do not use strings concatenation, especially in logger. Please use instead:
```python
logger.error("Couldn't find: %s", self.P['RoiFilesFolder'])
```